nodes/core/locales/en-US/messages.json

```diff
@@ -269,7 +269,7 @@
     },
     "mqtt": {
         "label": {
-            "broker": "Server",
+            "broker": "Server/URL",
```

Comment:
Concerned this will confuse the user if they then try other non wss:// urls. Suggest leaving it as `Server`.